app/services/proofing/mock/ddp_mock_client.rb

```diff
@@ -1,27 +1,38 @@
 module Proofing
   module Mock
-    class DdpMockClient < Proofing::Base
-      vendor_name 'DdpMock'
+    class DdpMockClient
+      class << self
```

Comment:
Why did `vendor_name`, `required_attributes`, `optional_attributes`, and `stage` become class methods?

Reply:
After looking over the old `Proofing::Base` I _mostly_ understand this now. 👍